grpc/src/main/java/io/stargate/grpc/service/ExceptionHandler.java

```diff
@@ -61,8 +61,10 @@ public void handleException(Throwable throwable) {
       handleException(throwable.getCause());
     } else if (throwable instanceof StatusException
         || throwable instanceof StatusRuntimeException) {
+      LOG.error(getErrorMessage(null, throwable));
```

Comment:
we should always include the exception in the log:

`LOG.error(message, exception)`